easybuild/easyconfigs/a/Autotools/Autotools-20150119-intel-2015a.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'Bundle'
+
+name = 'Autotools'
+version = '20150119' # date of the most recent change
+
+homepage = 'http://autotools.io'
+description = """This bundle collect the standard GNU build tools: Autoconf, Automake and libtool"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
```

Comment:
This is more about keeping the dependency specifications free of hardcoded subtoolchains.

@ocaisa is preparing for the time when https://github.com/hpcugent/easybuild-framework/pull/1306 is ready and merged, after which subtoolchain specs should become useless (and even harmful)